go/vt/vttablet/onlineddl/vrepl_test.go

```diff
@@ -4,3 +4,224 @@
 */
```

Comment:
Can we remove this copyright at this point?

Either way, we should also add the vitess copyright.

Reply:
Good catch. For this file the copyright should be just vitess. There is not derivative work from `freno` here.

Reply:
fixed.